src/utils/logout.js

```diff
@@ -1,5 +1,6 @@
+////TO-DO: Implement Auth0
 const handleLogout = () => {
-  localStorage.removeItem('okta-token-storage');
+  localStorage.removeItem('Auth0-token-storage-stub');
```

Comment:
Because the logout function will be provided via `useAuth0()` (hook provided by Auth0), you'll have to call it directly in the component and this utility function won't really work.

Reply:
Okay, great